app/model/table/ComputerTable.scala

```diff
@@ -16,7 +16,7 @@ class ComputerTable(tag: Tag) extends Table[Computer](tag, "computer") {
   def name = column[Option[String]]("name")
 
   // Room foreign key
-  def room: ForeignKeyQuery[RoomTable, Room] = foreignKey("room_id", roomId, TableQuery[RoomTable])(_.id, onUpdate = ForeignKeyAction.Restrict, onDelete = ForeignKeyAction.Cascade)
+  def room: ForeignKeyQuery[RoomTable, Room] = foreignKey("room_id", roomId, TableQuery[RoomTable])(_.id?, onUpdate = ForeignKeyAction.Restrict, onDelete = ForeignKeyAction.Cascade)
```

Comment:
`?` is a good contribution, I read about optional column checking with `?` but I still don't figure how it does the check. 😕